sample/android/src/main/AndroidManifest.xml

```diff
@@ -4,22 +4,19 @@
     <uses-permission android:name="android.permission.INTERNET" />
 
     <application
-        android:name="com.mikepenz.markdown.ui.MyApplication"
         android:allowBackup="true"
         android:icon="@mipmap/ic_launcher"
         android:label="@string/app_name"
         android:supportsRtl="true"
-        android:theme="@style/Theme.Sample">
+        android:theme="@android:style/Theme.Material.Light.NoActionBar">
```

Comment:
can you please explain the replacement of the prior theme?

Reply:
Doing so removes the `androidx-material` dependency